front/app/containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm/index.tsx

```diff
@@ -84,16 +85,29 @@ const CustomPageSettingsForm = ({
   const areas = useAreas();
   const appConfig = useAppConfiguration();
   const locale = useLocale();
+  const configuredLocales = useAppConfigurationLocales();
   const topics = useTopics();
   const { formatMessage } = useIntl();
 
+  const hasMultipleConfiguredLocales = !isNilOrError(configuredLocales)
+    ? configuredLocales.length > 1
+    : false;
```

Comment:
This is not ideal. Better would be to have a hook, but I'm taking this to the FE CDM. I also have no time to work on this now. Just getting the bug fixed.